pkg/broker/ingress/handler.go

```diff
@@ -127,15 +136,26 @@ func (h *Handler) ServeHTTP(response nethttp.ResponseWriter, request *nethttp.Re
 
 	event.SetExtension(EventArrivalTime, cev2.Timestamp{Time: time.Now()})
 
+	ctx, span := trace.StartSpan(ctx, kntracing.BrokerMessagingDestination(broker))
+	defer span.End()
+	if span.IsRecordingEvents() {
+		span.AddAttributes(
+			kntracing.MessagingSystemAttribute,
+			tracing.PubSubProtocolAttribute,
+			kntracing.BrokerMessagingDestinationAttribute(broker),
```

Comment:
What does this attribute represent?

For example, if this attribute is on a span, how can I interpret that span?

I ask because it looks like in the ingress we are annotating the span that sends the request to GCP PubSub with the broker as the destination. While in the filtering code I think the annotation states the trigger name while sending to the trigger's destination.

Reply:
This attribute considers each broker or trigger to be an event destination. On ingress a single broker span will be created and on fanout multiple trigger spans will be created identifiable by the messaging.destination attribute as well as the span name.